src/app/child-dev-project/notes/notes-manager/notes-manager.component.ts

```diff
@@ -63,28 +64,27 @@ export class NotesManagerComponent implements OnInit {
     {
       key: "urgent",
       label: $localize`:Filter-option for notes:Urgent`,
-      filter: { "warningLevel.id": WarningLevel.URGENT },
+      filter: { "warningLevel.id": WarningLevel.URGENT } as DataFilter<any>,
     },
     {
       key: "follow-up",
       label: $localize`:Filter-option for notes:Needs Follow-Up`,
       filter: {
-        "warningLevel.id": { $in: [WarningLevel.URGENT, WarningLevel.WARNING] },
-      },
+        "warningLevel.id": { $in: [WarningLevel.WARNING] },
+      } as DataFilter<any>,
```

Comment:
@sleidig do you think its fine if we change the default behaviour (currently "needs follow up" shows both warning and urgent. Maybe the label don't fit anymore.

Reply:
The original concept was to show anything needing action (warn + urgent) or with the other only show the urgent. That still makes sense but will be very easy to get with the new multi-value filters, so this change looks good to me.

Reply:
Then we can also remove the prebuilt filter and only define this through the config.